wifi-subsys/components/uart/src/at_handler.c

```diff
@@ -0,0 +1,213 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "driver/gpio.h"
+#include "driver/uart.h"
+#include "esp_log.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "subsys_uart.h"
+
+#include "at_handler.h"
+#include "subsys_uart.h"
+#include "wifi.h"
+
+#define ARRAY_SIZE 11
+
+char *default_keys[ARRAY_SIZE] = {
+    "NVS_RST", "WIFI_OFF",  "WAP_SSID", "WAP_PASS",  "WAP_CHAN",  "WAP_AUTH",
+    "WIFI_ON", "WIFI_MODE", "WIFI_RST", "WSTA_SSID", "WSTA_PASS",
+};
+
+char *symbols[4] = {
+    "+",
+    "&",
+    "?",
+    "=",
+};
+
+int get_symbol(char *command, size_t len, char *output) {
+    for (size_t i = 0; i < len; i++) {
+        for (size_t j = 0; j < 4; j++) {
+            if (command[i] == symbols[j][0]) {
+                memcpy(output, symbols[j], 2);
+                return ESP_OK;
+            }
```

Comment:
With the before suggest
```suggestion
            if (command[i] == symbols[j]) {
                memcpy(output, symbols[j], 2);
                return ESP_OK;
            }
```